deeplake/core/chunk_engine.py

```diff
@@ -1010,12 +1011,11 @@ def _handle_tiled_sample(
         lengths,
     ):
         sample = samples[0]
-        if sample.is_first_write:
-            if register:
-                if start_chunk_row is not None:
-                    enc.register_samples(1)
-                else:
-                    enc_count[-1] += 1
+        if sample.is_first_write and register:
```

Comment:
can this be combined into a single if statement?

Reply:
on line 1015 we have if start_chunk_row is not None:

Can we do 

if sample.is_first_write and register and start_chunk_row: